tests/ga/test_multi_config_extension.py

```diff
@@ -284,6 +284,7 @@ def __setup_generic_test_env(self):
         third_ext = extension_emulator(name="OSTCExtensions.ExampleHandlerLinux.thirdExtension")
         fourth_ext = extension_emulator(name="Microsoft.Powershell.ExampleExtension")
 
+        # pylint: disable=contextmanager-generator-missing-cleanup
```

Comment:
this needs to be inline, on the line producing the warning

Reply:
also, if you can add a comment to each suppression pointing out what generator is triggering the warning, it'd be great